k8s/scripts/crio-extractor.sh

```diff
@@ -30,8 +30,10 @@ function install_all() {
 function install_cni() {
 	install ${SELINUX} -d -m 755 ${CNIDIR}
 	install ${SELINUX} -D -m 755 -t ${OPT_CNI_BIN_DIR} cni-plugins/*
-	cp contrib/10-crio-bridge.conf contrib/100-crio-bridge.conf
-	install ${SELINUX} -D -m 644 -t ${CNIDIR} contrib/100-crio-bridge.conf
+	if ! is_host_gke; then
```

Comment:
Out of curiosity, why was this change snippet necessary?

Reply:
Oh, actually it was not! I verified that file on disk in a new build and everything still worked great. All that's needed are the additional `crio.network.plugin_dirs`